gp_align/align.py

```diff
@@ -7,36 +26,40 @@
 
 
 def align_plates(plate_image, calibration_plate):
-    """Both images should be converted to edges
-    Returns a vector (x, y) that describes the translation from the calibration plate to the analyte plate"""
+    """
+    Both images should be converted to edges
+    Returns a vector (x, y) that describes the translation from the calibration
+    plate to the analyze plate.
+    """
     r = int(RADIUS)
     best_offset = (0, 0)
     best_value = 0
-    for i in np.linspace(-r, r, 2*r+1).astype(int):
-        for j in np.linspace(-r, r, 2*r+1).astype(int):
+    # TODO: Potentially use vectorized numpy functions here.
+    for i in np.arange(-r, r, 2 * r + 1):
```

Comment:
This is not working. To work with arange it should be `arange(-r, r+1)`

Reply:
I saw this last moment and apparently introduced a bug right away :)

Reply:
On second thought, is there any reason not to use the normal `range` function?

Reply:
I honestly don't remember why I went for linspace and type conversion. I don't see any reason why range would not work